web/src/components/molecules/Common/ShadowCard/index.tsx

```diff
@@ -0,0 +1,17 @@
+import Card from "@marketplace/components/atoms/Card";
+import { styled } from "@marketplace/theme";
+
+export type Props = {
+  children: React.ReactElement;
+};
+
+const ShadowCard: React.FC<Props> = ({ children }) => {
+  return <CustomCard>{children}</CustomCard>;
+};
+
+const CustomCard = styled(Card)`
+  background-color: transparent;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+`;
```

Comment:
```suggestion
const ShadowCard = styled(Card)`
  background-color: transparent;
  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
`;
```
Seems like we can use styled component directly here?

Reply:
Sweet. I didn't know that. Added